local-infrastructure/run.sh

```diff
@@ -44,13 +44,22 @@ docker logs -f comfort_node_migrate
 #
 docker exec comfort_keycloak bash /tmp/keycloak-local-user.sh
 
-# Build and serve in development mode
+# Build application
 #
 pushd ../app/
 npm run all:install
 npm run all:build
-npm run serve
 
-# Open app
+# Install open-cli and browse to site when app is available
+#
+[ -f ./node_modules/.bin/open-cli ] || npm install open-cli
+(
+    while (! curl -s http://localhost:8080 -o /dev/null); do
+        sleep 10
+    done
+    ./node_modules/.bin/open-cli http://localhost:8080/
+) &
```

Comment:
There is a way to avoid having to install open-cli entirely by leveraging npm's `npx` tool - it'll dynamically download and run a package on the fly.

`npx open-cli http://localhost:8080/` is the effective one-line command. Can probably simplify this to just do the while loop, and then execute the open-cli command without having to do the bash file check.

Reply:
@jujaga Good idea and done!  Thanks.  :)